### PR TITLE
romstage: fixed version string

### PR DESCRIPTION
**Pull request by miczyg1**
The version string is printed twice, because it is implemented in `mainboard.c` and in `romsatge.c`. Also the BIOS debug level should be changed. Do we want to keep debug levels as it is or should I change it?
Feel free to discuss, so I can change it eventually.

### CODE REVIEW

**Comment by mek-x** [comment]
@miczyg1 could you copy here, how the boot splash looks like now?

**Comment by miczyg1** [comment]
[bootsplash.log](https://github.com/pcengines/coreboot/files/1332184/bootsplash.log)
@mekx look at line 16 and 198. The debug level should be unified. I also have not changed the log level in config (SPEW currently).

**Comment by miczyg1** [comment]
I separated printing version info and memory info to romstage and mainboard as it is in mainline. I also changed the BIOS debug level to ALERT to prevent unnecessary output. I attach config and console output. Please @mek-x approve and close PR if possible.

[config-legacy.txt](https://github.com/pcengines/coreboot/files/1339896/config-legacy.txt)
[version-string-legacy.log](https://github.com/pcengines/coreboot/files/1339898/version-string-legacy.log)

File: src/include/version.h [hunk 1]
```diff
@@ -14,9 +14,6 @@ extern const unsigned int coreboot_version_timestamp;
 
 /* When coreboot was compiled */
 extern const char coreboot_compile_time[];
-extern const char coreboot_compile_by[];
```
Comment (mek-x) [inline src/include/version.h]: @miczyg1 those edits are unnecessary. I know those vars are not used, but do not change them without a valid reason. Anyway this is legacy version - let's keep the changes to necessary minimum.

File: src/lib/coreboot_table.c [hunk 2]
```diff
@@ -282,9 +282,6 @@ static void lb_strings(struct lb_header *header)
 		{ LB_TAG_EXTRA_VERSION,  coreboot_extra_version,  },
 		{ LB_TAG_BUILD,          coreboot_build,          },
 		{ LB_TAG_COMPILE_TIME,   coreboot_compile_time,   },
-		{ LB_TAG_COMPILE_BY,     coreboot_compile_by,     },
```
Comment (mek-x) [inline src/lib/coreboot_table.c]: As above

File: src/lib/version.c [hunk 3]
```diff
@@ -18,12 +18,6 @@
 #ifndef COREBOOT_COMPILE_TIME
 #error  COREBOOT_COMPILE_TIME not defined
 #endif
-#ifndef COREBOOT_COMPILE_BY
```
Comment (mek-x) [inline src/lib/version.c]: As above

File: src/lib/version.c [hunk 4]
```diff
@@ -43,9 +37,6 @@ const char coreboot_build[] = COREBOOT_BUILD;
 const unsigned int coreboot_version_timestamp = COREBOOT_VERSION_TIMESTAMP;
 
 const char coreboot_compile_time[]   = COREBOOT_COMPILE_TIME;
-const char coreboot_compile_by[]     = COREBOOT_COMPILE_BY;
```
Comment (mek-x) [inline src/lib/version.c]: As above

**Pull request closed without merging**